pspgl/PSPBUILD

```diff
@@ -0,0 +1,50 @@
+# Maintainer: Wouter Wijsman <wwijsman@live.nl>
+pkgname=pspgl
+pkgver=r59.2e82204
+pkgrel=1
+pkgdesc="OpenGL-ESish library for PSP"
+arch=('mips')
+url="https://github.com/pspdev/psp-ports"
+license=('BSD')
+depends=()
+makedepends=()
+optdepends=()
+source=("git+https://github.com/pspdev/psp-ports.git")
+sha256sums=('SKIP')
+
+pkgver() {
+  cd psp-ports
+  # Set the revisioin of the repo as version
+  # Means any update to psp-ports will result in a new version number
+  printf "r%s.%s" "$(git rev-list --count HEAD)" "$(git rev-parse --short HEAD)"
+}
+
+prepare() {
+	cd "psp-ports/$pkgname"
+	for i in glLockArraysEXT pspgl_dlist pspgl_vidmem
+	do
+		echo '#include <psptypes.h>' > $i.c.tmp
```

Comment:
This is ugly. Why not apply in the psp-port instead?

Reply:
I was wondering this too, since it's in the build script in psp-ports. I can fix this later.

Reply:
I've made the following PRs to address this:
- https://github.com/pspdev/psplibraries/pull/67
- https://github.com/pspdev/psp-ports/pull/33

When these have been accepted, this code should no longer be needed. Do merge both of them at the same time, though, otherwise there will be issues.

Reply:
The mentioned PRs have been merged and the code removed.